<launch>

    <node name="pcl_coloriser_node" pkg="myproject1" type="pcl_coloriser_node" output="screen">

        <param name="op_path" value="/home/srbh/agrirobo_proj/with_pcls/data/colored/" />
        <param name="save_flag" value="true" />

        <remap from="input_cloud" to="/sensor/laser/vlp16/front/integrated_pointcloud_xyzi"/>
        <remap from="camera_info" to="/sensor/camera/blackfly/camera_info"/>
        <remap from="input_image" to="/sensor/camera/blackfly/image_rect_color"/>

        <remap from="output_cloud" to="/sensor/laser/vlp16/front/colored_pointcloud_xyzirgbt"/>
    </node>
</launch>